src/actions/auth.js

```diff
@@ -0,0 +1,78 @@
+const SUCCESS_STATUS = 200;
+const UNAUTHORIZED_STATUS = 401;
```

Comment:
It would be better off making a separate file for the constants.

Reply:
Thanks.  I'm eager to see your new unquestionably useful and thought-provoking comments!